recipes/emperator/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "emperator" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3ba756b2ca40b5886162affced4f7ce2a992227d030699c454a529f3b61de2cb
+
+build:
+  skip: true  # [win]
```

Comment:
can this be noarch? It's OK if Windows fails when the dependency is missing.

Reply:
> It's OK if Windows fails when the dependency is missing.

You mean it can be noarch even if the emsdk Windows package is not existent?

Reply:
I just pushed a commit to make it noarch